e2e/client/playwright/page-object-models/monitoring.ts

```diff
@@ -38,6 +38,29 @@ export class Monitoring {
             .click();
     }
 
+    async executeActionInEditor(...actionPath: Array<string>): Promise<void> {
+        await this.page.locator(s('authoring-topbar', 'actions-button')).click();
+
+        const actionsWithoutLast = actionPath.slice(0, actionPath.length - 1);
+
+        for (const action of actionsWithoutLast) {
+            await this.page.locator(s('actions-list')).getByRole('button', {name: action}).hover();
+        }
+
+        await this.page.locator(s('actions-list'))
+            .getByRole('button', {name: actionPath[actionPath.length - 1]})
+            .click();
+    }
+
+    async executeMultiAction(selectedArticles: Array<string>, action: string): Promise<void> {
```

Comment:
adjust function signature to:
`executeBulkAction(action: string, articleNames: Array<string>): Promise<void>`